src/assets/purple-blast.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="41" height="41" viewBox="0 0 41 41" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
These SVGs are images and not paths. Will we have the path versions of it?

Reply:
No but lets not worry about that for now design needs to export them

Reply:
But yes they are images and hopefully im not sure whats the status of them waiting on design but dont want to hang on this